recipes/ptitprince/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ptitprince" %}
+{% set version = "0.2.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ptitprince-{{ version }}.tar.gz
+  sha256: 1b3dbac736f1180d865b9ae2508f2ce59769fc85800cbee120d2a3cbc67c6368
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - cython
+    - matplotlib-base
+    - numpy >=1.13
+    - pyhamcrest >=1.9.0
+    - python
+    - scipy
+    - seaborn >=0.10
+
+test:
+  imports:
+    - ptitprince
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/pog87/PtitPrince
+  summary: A Python implementation of Rainclouds, originally on R, ggplot2. Written on top of seaborn.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - pog87
```

Comment:
@pog87 are you OK being a maintainer here?

Reply:
@ocefpaf I am ok